client/src/pages/SignUp/useStyles.ts

```diff
@@ -14,6 +14,9 @@ const useStyles = makeStyles(() => ({
     flexDirection: 'column',
     minHeight: '100vh',
     paddingTop: 23,
+    '& .makeStyles-authHeader-6': {
+      justifyContent: 'center',
+    },
```

Comment:
Same here, also, another point. If the styling is nearly identical between both pages, we probably could just re-use the components for both login and sign up?

Reply:
removed, now one style in AuthHeader to center the auth header in smaller screen is being applied  in both pages
![1DDE30D0-30E5-4E10-9722-E284B2B5639A](https://user-images.githubusercontent.com/19751790/128306753-304a9428-f1d0-488e-9ada-ca02fb7146e3.jpeg)